src/pages/App.js

```diff
@@ -430,24 +423,6 @@ class App extends Component {
     this.setState({ showWeb3ErrorBanner: false })
   }
 
-  static getDerivedStateFromError(_) {
-    return { didError: true }
-  }
-
-  componentDidCatch(error, errorInfo) {
-    try {
-      Sentry.withScope(scope => {
-        scope.setExtras(errorInfo)
-        Sentry.captureException(error)
```

Comment:
Worried that getting rid of this might break alerts, as the saga's `Sentry.captureException(new Error(action.message))` seems different than this? Maybe we do this code in the boundary to keep it consistent rather than dispatching the action for the saga.

Reply:
I see, is there a way to update the saga part to be consistent, or do you think that these call for different sentry implementations?

Reply:
If they're different now, I think making it consistent will break the tracking/bucketing we have in sentry, but I'm not sure. CC: @raymondjacobson ? It might mean that besides making it consistent we'll also need to check and update sentry things?

Reply:
@rickyrombo i think this might be the same actually. so the setExtras comes from errorInfo, which gets passed to `handleError` with `additionalInfo: errorInfo`, and the `captureException` gets passed correctly with `error.message`. am i missing something?

Reply:
They could very well be effectively the same, not super familiar with the Sentry API. I do see that in the sagas we call:
```ts
        Sentry.captureException(new Error(action.message))
```
which seems different than capturing the error object?

Reply:
Like with your change, instead of `Sentry.captureException(error)` it's essentially `Sentry.captureException(new Error(error.message))` which may or may not be different?

Reply:
right, and from what i can tell that is the same, as long as the error class is the same, which it is